crates/assistant/Cargo.toml

```diff
@@ -43,6 +44,9 @@ ui.workspace = true
 util.workspace = true
 uuid.workspace = true
 workspace.workspace = true
+reqwest = { version = "0.11", features = ["blocking"] }
```

Comment:
We already have an HTTP client in Zed that we should be using instead of `reqwest`.

Reply:
Sure. Noted. Thank you for suggestion.